### PR TITLE
pallets: add migration to trigger #676 fix

### PR DESCRIPTION
**Pull request by h4x3rotab**
- add migration to trigger #676 fix
- remove out-of-date migrations

### CODE REVIEW

File: pallets/phala/src/migrations/mod.rs [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+	type Versions = (
+		StorageVersion,
+		StorageVersion,
+		StorageVersion,
+		StorageVersion,
+		StorageVersion,
+	);
+
+	const EXPECTED_STORAGE_VERSION: Versions = (
```
Comment (jasl) [inline pallets/phala/src/migrations/mod.rs:27]: I prefer struct than tuple, struct with field name can self-describe.

I was thinking there're 6 pallets above but here's only 5 and confuse the order

Comment (jasl) [reply pallets/phala/src/migrations/mod.rs:27]: ignore me if this one not reuse in future

File: crates/phala-types/src/lib.rs [hunk 2]
```diff
@@ -512,7 +512,9 @@ pub mod messaging {
     pub enum GatekeeperEvent {
         NewRandomNumber(RandomNumberEvent),
         TokenomicParametersChanged(TokenomicParameters),
+        /// Deprecated after https://github.com/Phala-Network/phala-blockchain/pull/499
         RepairV,
```
Comment (jasl) [inline crates/phala-types/src/lib.rs:516]: I suggest we add `_` prefix for depracated enum like `_RepairV` which more self-explanation

File: pallets/phala/src/migrations/mod.rs (hunk 1 above)
Comment (h4x3rotab) [reply pallets/phala/src/migrations/mod.rs:27]: Don't bother make it clear. In the future all the storage version will bump together

File: crates/phala-types/src/lib.rs (hunk 2 above)
Comment (h4x3rotab) [reply crates/phala-types/src/lib.rs:516]: It's still in used in the code, but just shouldn't be triggered in the future

**Pull request merged**